spec/models/appointment_spec.rb

```diff
@@ -11,6 +11,42 @@
     expect(subject.reference).to eq(subject.booking_request.reference)
   end
 
+  describe '#fulfilment_time_seconds' do
+    subject { build(:appointment) }
+
+    it 'defaults to 0' do
+      expect(described_class.new.fulfilment_time_seconds).to be_zero
```

Comment:
I take it you can tell which records have a valid value as they have a `proceeded_at` time rather than using a `nil` default value to determine this?